### DISCUSSION TITLE
Insights

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#20] [Q&A]
@Sensinki Kan je de insights omzetten naar alle branches ipv van alleen de main? Idk of het kan, maar zou je kunnen kijken als je tijd hebt :)

**Comment by Sensinki** [comment]
Ik heb een beetje bekeken maar ik denkt dat we alleen main branch kunnen zien.

**Reply by D0nnaz** [reply]
Alright, thanks voor het kijken! Is niet super belangrijk

**Comment by D0nnaz** [comment]
Duidelijk
